models/autoencoders/autoencoderbase.py

```diff
@@ -38,6 +38,9 @@ def __init__(self, flat_input_size, train_ds_mean, activation_fct, cfg, **kwargs
 
         # the main configuration namespace returned by configaro
         self._config=cfg
+        
```

Comment:
Hey @abhishekabhishek, it's a minor thing, but not all models inheriting from autoencoderbase need this variable. So I would prefer to keep it in the Discrete VAE models - afaik the `DiVAEpp` and `Gumbolt` models inherit from `DiVAE`, so we could put it there.

Reply:
i believe it already existed in autoencoderbase. i did not add this here.

Reply:
Thanks!

Reply:
@abhishekabhishek I removed this in c4fb11d3ee36888638bebd6071ed367888e8794e and replaced it with the corresponding member var from the `DiVAE` model.